bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/ICloudService.java

```diff
@@ -246,8 +246,13 @@ public boolean validate2faCode(String code) throws IOException, InterruptedExcep
             this.session.post(AUTH_ENDPOINT + "/verify/trusteddevice/securitycode", JsonUtils.toJson(requestBody),
                     headers);
         } catch (ICloudApiResponseException ex) {
-            logger.debug("Code verification failed.", ex);
-            return false;
+            this.logger.debug("Exception on code verification. Verification might still be successful.", ex);
```

Comment:
The idea was to log `ex.getStatusCode()`, not the full stack trace. So maybe something like
```
logger.debug("Exception on code verification with HTTP status {}. Verification might still be successful.", ex.getStatusCode());
```

Reply:
If in addition to the status code, the stack trace would be helpful, I would suggest to rather use TRACE level:
````
logger.trace("Exception on code verification with HTTP status {}. Verification might still be successful.", ex.getStatusCode(), ex);
```

Reply:
The full exception is logged in line 249. I avoided logging it twice.

Reply:
I do not understand your comment. Line 249 is the line I am pointing.

Our request was probably not enough clear. The proposition was to log systematically the status code so that if one day another status code is not caught by your following test, you could easily know what status code is not covered by your test.
Currently you are logging the full stack trace exception, that is even not the data we want to log.
Stack trace is probably even not necessary.

Reply:
No worries, but I still don't get why it is a problem to have the stacktrace on DEBUG, but ok it's easy to change that.